spec/features/guests/show_spec.rb

```diff
@@ -0,0 +1,36 @@
+require "rails_helper"
+
+
+RSpec.describe "guest show page" do
+  before(:each) do
+    @powder = Hotel.create!(name: "Powder Palace", location: "Powder mountain")
+    @eagle = Hotel.create!(name: "Eagle's Nest", location: "Eagle mountain")
+
+    @room_1p = @powder.rooms.create!(suite: "Diamond", rate: 5000)
+    @room_2p = @powder.rooms.create!(suite: "Gold", rate: 3000)
+    @room_3p = @powder.rooms.create!(suite: "Silver", rate: 2000)
+
+    @guest_1 = @room_1p.guests.create!(name: "Margo Robbie", nights: 3)
+    @guest_2 = @room_2p.guests.create!(name: "Margo Robbie", nights: 5)
+    @guest_3 = @room_1p.guests.create!(name: "Margo Robbie", nights: 2)
+    @guest_4 = @room_3p.guests.create!(name: "Kevin Bacon", nights: 1)
+    @guest_5 = @room_2p.guests.create!(name: "Kevin Bacon", nights: 3)
+    @guest_6 = @room_3p.guests.create!(name: "Kevin Bacon", nights: 2)
```

Comment:
Good object setup here!

Reply:
For this first story, it might be a little _too many_ objects than what you need to get started, I would say try to stick to 2-3 of each type of object, which should be plenty for most things we ask for in ICs.